lib/filesystem/src/filesystem.cpp

```diff
@@ -592,15 +560,11 @@ namespace irods::experimental::filesystem::NAMESPACE_IMPL
     {
         const auto s = status(_comm, _p);
 
-        if (is_data_object(s)) {
-            return data_object_size(_comm, _p) == 0;
-        }
-
         if (is_collection(s)) {
             return is_collection_empty(_comm, _p);
         }
 
-        throw filesystem_error{"cannot check emptiness: unknown object type", _p, make_error_code(CAT_NOT_A_DATAOBJ_AND_NOT_A_COLLECTION)};
+        throw filesystem_error{"cannot check emptiness: object type unsupported", _p, make_error_code(CAT_NOT_A_DATAOBJ_AND_NOT_A_COLLECTION)};
```

Comment:
there are other places in this file where **unknown object type** is used.  either the other references need to be updated to use the same phrase or this should be restored to fall in line with the others.

Reply:
In this case, this code will be restored along with the data_object_size family of functionality, so I will just revert these pieces of the change.